chapters/chapter4.html

```diff
@@ -33,10 +33,10 @@ <h2>Terminology</h2>
           <section>
             <h2>Grid Container</h2>
             <p>
-              The container that holds the entire CSS grid. It is the element that has the
-              display: <span class="code">grid</span>
-              or
-              display: <span class="code">inline-grid</span>
+              The container that holds the entire CSS grid. It is the element that has the display:
+              <span class="code">grid</span>
+              or display:
+              <span class="code">inline-grid</span>
               on it.
```

Comment:
I stumbled a bit over the phrasing "the display: code on it". Do we maybe want to add "property" here? e.g. "that has the display: grid or display: inline-grid property on it"